### PR TITLE
Add toast countdown

### PR DESCRIPTION
**Pull request by SAnatoliiS**

### CODE REVIEW

File: packages/popups/src/Notice/Notice.tsx [hunk 1]
```diff
@@ -7,34 +7,50 @@ import {
...
+import { CountdownCirlce } from './CountdownCircle';
 
 const getBackgroundColor = (color: UINoticeColor, theme: Theme): ColorValue => {
     switch (color) {
         case UINoticeColor.Secondary:
             return theme[ColorVariants.BackgroundSecondary];
         case UINoticeColor.Negative:
-            return theme[ColorVariants.BackgroundNegative];
+            /**
+             * TODO Add new color to ColorVariants
```
Comment (savelichalex) [inline packages/popups/src/Notice/Notice.tsx]: Will you add it? :)

File: packages/popups/src/Notice/Notice.tsx [hunk 2]
```diff
@@ -52,42 +68,51 @@ export const Notice: React.FC<NoticeProps> = ({
...
+                    onLongPress={onLongPress}
+                    delayLongPress={UIConstant.notice.longPressDelay}
+                    onPressOut={onPressOut}
                 >
-                    {title}
-                </UILabel>
+                    <View style={StyleSheet.absoluteFill} />
+                </TouchableWithoutFeedback>
+                <View style={styles.countdown}>
+                    <CountdownCirlce
```
Comment (savelichalex) [inline packages/popups/src/Notice/Notice.tsx]: So countdown isn't opt-in? 🤔

Comment (SAnatoliiS) [reply packages/popups/src/Notice/Notice.tsx]: Of course! I forgot to add the prop.

File: packages/popups/src/Notice/Notice.tsx (hunk 1 above)
Comment (SAnatoliiS) [reply packages/popups/src/Notice/Notice.tsx]: When the designer adds this color to the palette

Comment (savelichalex) [reply packages/popups/src/Notice/Notice.tsx]: Can you please discuss it with Aleksei and let's add it now, no need to wait for it

Comment (SAnatoliiS) [reply packages/popups/src/Notice/Notice.tsx]: Done

File: packages/popups/src/Notice/types.ts [hunk 3]
```diff
@@ -34,11 +33,15 @@ export type NoticeProps = {
     onLongPress: () => void;
     onPressOut: () => void;
     action?: UINoticeActionAttributes;
+    countdownValue: Animated.SharedValue<number>;
+    countdownProgress: Animated.SharedValue<number>;
+    isCountdownVisible?: boolean;
```
Comment (savelichalex) [inline packages/popups/src/Notice/types.ts]: What do you think to call it `hasCountdown` ? It's shorter :)

**Review by savelichalex — APPROVED** [review]

File: packages/popups/src/Notice/CountdownCircle.tsx [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+         * Used for web
+         */
+        'stroke-dashoffset': true,
+    }),
+);
+
+const circleOffset = 0.5;
+
+const size = 20;
+const strokeWidth = 1.2;
```
Comment (sertony) [inline packages/popups/src/Notice/CountdownCircle.tsx]: Why not moving it Constants?

**Review by sertony — APPROVED** [review]

**Pull request merged**